common/lib/xmodule/xmodule/js/src/video/04_video_full_screen.js

```diff
@@ -5,11 +5,9 @@ define('video/04_video_full_screen.js', [], function () {
         '<button class="control add-fullscreen" aria-disabled="false" title="',
             gettext('Fill browser'),
         '">',
-            '<span class="icon-fallback-img">',
-                '<span class="icon fa fa-arrows-alt" aria-hidden="true"></span>',
-                '<span class="sr control-text">',
-                    gettext('Fill browser'),
-                '</span>',
+            '<span class="icon fa fa-arrows-alt" aria-hidden="true"></span>',
+            '<span class="sr control-text">',
```

Comment:
Since this button has a `title` we can simplify this and remove the `sr` text within it.